Testing/SystemTests/tests/framework/ISIS/SANS/WORKFLOWS/SANSBatchReductionTest.py

```diff
@@ -6,9 +6,11 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 # pylint: disable=too-many-public-methods, invalid-name, too-many-arguments
 import unittest
+import os
 
 import systemtesting
 
+import mantid
```

Comment:
Is it possible to just import the config from mantid (e.g. `from mantid import config`)?